Portage/Dependency/Print.hs

```diff
@@ -15,6 +17,10 @@ import Text.PrettyPrint ( (<>), vcat, nest, render )
 
 import Portage.Dependency.Types
 
+#if MIN_VERSION_base(4,11,0)
+import Prelude hiding ((<>))
+#endif
```

Comment:
I suggest importing qualified (<>) from `Text.PrettyPrint` explicitly as:
```haskell
import Text.PrettyPrint as PP ((<>))
```
and switch to PP.<> in code to simplify reading of the code.